### PR TITLE
Create project folder for layered architecture theme  + README file

### PR DESCRIPTION
**Pull request by anamalaiaEndava**

### CODE REVIEW

**Comment by endalexei** [comment]
could you please delete toBeDeleted.file as it was used to create folders

File: ST/30_layered_architecture/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+## Basic idea
+***
+The most common architecture pattern is the layered architecture pattern, otherwise known as the n-tier architecture pattern. 
+This pattern is the de facto standard for most Java EE applications and therefore is widely known by most architects, designers, and developers. 
+
+Components within the layered architecture pattern are organized into horizontal layers, each layer performing a
+specific role within the application. Request passes through layers from top to bottom and then back up,
+and each layer can depend on one or a few layers below it but should not know anything about any layer above it
+<p align="center" width="100%">
+    <img width="60%" src=layered%20architecture.png>
```
Comment (endalexei) [inline ST/30_layered_architecture/README.md:21]: it didnt load for me, the other one did :)n could you check that ?

**Review by endalexei — APPROVED** [review]

File: ST/30_layered_architecture/README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+# Layered (N-tier) architecture
```
Comment (bodacd) [inline ST/30_layered_architecture/README.md:1]: Could you please rename README file to something more related for Layerd Architecture?
For example `Layerd_Architecture_README.md`

Comment (anamalaiaEndava) [reply ST/30_layered_architecture/README.md]: Thanks, renamed

**Review by bodacd — APPROVED** [review]

**Pull request merged**